### PR TITLE
fix(curriculum): description change of learn-css-variables-by-building-a-city-skyline

### LINKED ISSUE #47555: feedback on Learn Css Variables By Building A City Skyline
### Describe the Issue

this project assumes you haven't worked with Flexbox yet hence it just givesyou the code without it being a real challenge/test.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-css-variables-by-building-a-city-skyline

### Your code

```
none
```

### Expected behavior

just to be given directions about what to do (as per usual) without giving you all the lines of code.

### Screenshots

Screenshot at 2022-09-18 15-46-28

### System

- Device: Zenbook
- OS: Linux Mint 20.3
- Browser: Brave
- Version: v1.43.93

### Additional context

_No response_

### LINKED ISSUE #47740: Mismatch between text and practice projects sequence
https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98d5.md

In this step it says 
> You will learn about [flexbox] in more detail on another project.

but the flexbox practice project comes before this one. This has to be removed or changed to indicate a previosu project.

### PR DESCRIPTION
**Pull request by nayabatir1**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #47555

`learn-css-variables-by-building-a-city-skyline` description changed as per comment

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98d5.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ dashedName: step-13
 
 # --description--
 
-Give your `.bb1` element these style properties: `display: flex;`, `flex-direction: column;`, and `align-items: center;`. This will center the parts of the building using "flex" or "flexbox". You will learn about it in more detail on another project.
+Try to center your `.bb1` element, using these style properties: `display: flex;`, `flex-direction: column;`, and `align-items: center;`.
```
Comment (jeremylt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98d5.md:10]:
```suggestion
Center your `.bb1` element by using these style properties: `display: flex;`, `flex-direction: column;`, and `align-items: center;`.
```

**Comment by moT01** [comment]
There's a [duplicate of the issue with some more suggestions](https://github.com/freeCodeCamp/freeCodeCamp/pull/47796) I like a little better. "Center the parts of your building by turning the .bb1 element into a flexbox parent. Use the flex-direction and align-items properties to center the children." is the example I gave there.

**Comment by Sboonny** [comment]
I agree with Tom, again sorry @nayabatir1 for the confusion

**Comment by ShaunSHamilton** [comment]
Hello there,

Please edit this PR title to match our recommended title format: https://contribute.freecodecamp.org/#/how-to-open-a-pull-request

**Comment by nayabatir1** [comment]
> Hello there,
> 
> Please edit this PR title to match our recommended title format: https://contribute.freecodecamp.org/#/how-to-open-a-pull-request

Done

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
> Done

This is not a documentation change... the automatically applied label https://github.com/freeCodeCamp/freeCodeCamp/labels/scope%3A%20curriculum could be an hint toward the scope

Please consult our [contributing documentation](https://contribute.freecodecamp.org/#/), specifically [How to open a Pull Request (PR)](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=how-to-open-a-pull-request-pr)

**Comment by nayabatir1** [comment]
> > Done
> 
> This is not a documentation change... the automatically applied label [ scope: curriculum](https://github.com/freeCodeCamp/freeCodeCamp/labels/scope%3A%20curriculum) Lessons, Challenges, Projects and other Curricular Content in curriculum directory. could be an hint toward the scope
> 
> Please consult our [contributing documentation](https://contribute.freecodecamp.org/#/), specifically [How to open a Pull Request (PR)](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=how-to-open-a-pull-request-pr)

Updated

**Comment by ieahleen** [comment]
The linked issue is still with https://github.com/freeCodeCamp/freeCodeCamp/labels/scope%3A%20curriculum and has not been opened to contributions, as such I am blocking this until consense ahs been reached.

**Comment by nayabatir1** [comment]
Hey @ieahleen is there anything that is pending from my end, do let me know as I'm not clear what needs to be done here.

**Comment by ieahleen** [comment]
You have opened the PR prematurely, the issue was not open for contributions and is still being discussed

**Pull request closed without merging**